README.md

```diff
@@ -1,2 +0,0 @@
-# development-platforms
```

Comment:
Please un-delete this file. The only change needed is your markdown file.

Reply:
Same with the other deleted files, you can add them back from the main branch and commit :)